lib/xlsx/xlsx.js

```diff
@@ -283,11 +284,27 @@ class XLSX {
         if (entryName[0] === '/') {
           entryName = entryName.substr(1);
         }
-        const stream = new PassThrough();
-        if (entryName.match(/xl\/media\//)) {
+        let stream;
+        if (entryName.match(/xl\/media\//) ||
+          // themes are not parsed as stream
+          entryName.match(/xl\/theme\/([a-zA-Z0-9]+)[.]xml/)) {
+          stream = new PassThrough();
           stream.write(await entry.async('nodebuffer'));
         } else {
-          const content = await entry.async('string');
+          // use object mode to avoid buffer-string convention
+          stream = new PassThrough({
+            writableObjectMode: true,
+            readableObjectMode: true,
+          });
+          let content;
+          // https://www.npmjs.com/package/process
+          if (process.browser) {
```

Comment:
should this be `if (!process.browser)` or am I misunderstanding the intention? `process.browser` is truthy only in browser environments, right?

Reply:
Yes, you are right. I'll rebase and fix that.